projects/packages/woocommerce-analytics/changelog/add-clickhouse-param

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Since you defined this change as "minor", when we release a new version of the package, a minor bump will happen. Since the current stable is `0.4.1`, that means it will bump to `0.5.0`.

That is perfectly fine, but I wanted to point that out since I see you're aiming to merge this PR into a `release/woocommerce-analytics-0.4.2` instead of `trunk`.

I should say you don't have to work in a release branch like this ; you could merge each one of your PRs into `trunk`, as long as those changes do not break anything in`trunk` of course. :)

Reply:
> Since you defined this change as "minor", when we release a new version of the package, a minor bump will happen. Since the current stable is 0.4.1, that means it will bump to 0.5.0.

Thanks @jeherve I didn't notice that.

> I should say you don't have to work in a release branch like this ; you could merge each one of your PRs into trunk, as long as those changes do not break anything intrunk of course. :)

YEah. not like breaking anything but sine the last incident with cookies I prefer to work in releases.